recipe/run_test.py

```diff
@@ -1,72 +1,12 @@
-from pandas._testing import assert_frame_equal
-import pandas as pd
+#!/bin/bash
+
```

Comment:
Currently we have the tests under run_test.py for validating pandas functionality. we no need to replace them the commands ur using u can append to the current test file or create these steps in the commands section.